.travis.yml

```diff
@@ -37,6 +37,10 @@ before_install:
   - sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable -y
   - sudo apt-get update -q
   - sudo apt-get install binutils libproj-dev gdal-bin -y
+  - sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/12/main/postgresql.conf
+  - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/12/main/pg_hba.conf
+  - sudo systemctl restart postgresql@12-main
+  - sudo cat /var/log/postgresql/postgresql-12-main.log
```

Comment:
Could you please remove this stuff, it would be not required after https://github.com/openwisp/django-rest-framework-gis/pull/248